components/bl-transitions-component/src/collapse-top.js

```diff
@@ -41,7 +39,7 @@ export function CollapseTop({ component, eventHandlers, transitionsContainerPod,
       <div
         ref={ rootRef }
         className={ cn('transition', variants, { [variants + '--active']: isTransition }) }
-        style={{ ...style, transitionDuration: duration + 'ms' }}>
+        style={{ ...style, transitionDuration: duration + 'ms', position: 'absolute', zIndex: -1, opacity: 0 }}>
```

Comment:
Same here, could it come from styles?

If yes - don't forget about variables.

Reply:
If no - create constant showElementStyles / hideElementStyles(or something like that) and use destruction here.

Reply:
> If yes - don't forget about variables.

I think that there is no need to add variables in the styles, because if you change these properties, the animation may not work correctly